### ISSUE TITLE
OSX: Yosemite requires "SDL/SDL_mixer.h" in includes again

### ISSUE THREAD

**Issue by sneakernets** [chocolate-doom/chocolate-doom#496]
This pops up after configure and make:

```
opl_sdl.c:26:10: fatal error: 'SDL_mixer.h' file not found
#include "SDL_mixer.h"
         ^
1 error generated.
make[3]: *** [opl_sdl.o] Error 1
make[2]: *** [all-recursive] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```
changing these to `SDL/file` works.

**Comment by sneakernets** [comment]
Closing this one because it's not an issue! This was user error.

**Comment by sneakernets** [comment]
Well, looks like I'm wrong, this is back as of the latest release. This may be because SDL_Mixer does NOT provide `pkg-config` files in its OS X releases. even if I compile from source, these are not found unless I add `SDL/` again.

**Comment by fragglet** [comment]
Works for me. How are you building?

**Comment by fabiangreffrath** [comment]
> This may be because SDL_Mixer does NOT provide pkg-config files in its OS X releases.

But all they do is give you an additional "-lSDL_mixer" for the "--libs" option. Everything else is the same as for "pkg-config sdl".

**Comment by fragglet** [comment]
I was under the impression that SDL_mixer v1.2.* doesn't even use pkg-config, hence my question.

**Comment by fabiangreffrath** [comment]
I think they started shipping them most recently in 1.2.11.

**Comment by sneakernets** [comment]
I'm using Homebrew to install SDL and the associated libraries. If it helps, Eternity's build system can detect those with no issues.

**Comment by fragglet** [comment]
You're probably doing something wrong then. The build-chocolate-doom script works fine.

Suffice it to say that #include "SDL/SDL_mixer.h" is incorrect.

**Comment by sneakernets** [comment]
I'm not using the build-chocolate-doom script though -- I'm building directly from the github source.

**Comment by jmtd** [comment]
OS X, for me, with SDL via brew:
```
0:tharsis$ brew list |grep sdl
sdl
sdl_mixer
0:tharsis$ pkg-config --libs --cflags sdl
-D_GNU_SOURCE=1 -D_THREAD_SAFE -I/usr/local/include/SDL -L/usr/local/lib -lSDLmain -lSDL -Wl,-framework,Cocoa
1:tharsis$ pkg-config --libs --cflags SDL_mixer
-D_GNU_SOURCE=1 -D_THREAD_SAFE -I/usr/local/include/SDL -L/usr/local/lib -lSDL_mixer -lSDLmain -lSDL -Wl,-framework,Cocoa
0:tharsis$ brew list sdl_mixer
/usr/local/Cellar/sdl_mixer/1.2.12/include/SDL/SDL_mixer.h
/usr/local/Cellar/sdl_mixer/1.2.12/lib/libSDL_mixer-1.2.0.dylib
/usr/local/Cellar/sdl_mixer/1.2.12/lib/pkgconfig/SDL_mixer.pc
/usr/local/Cellar/sdl_mixer/1.2.12/lib/ (2 other files)
0:tharsis$ ls -l /usr/local/include/SDL/SDL_mixer.h
lrwxr-xr-x  1 jon  admin  53 23 Apr 16:31 /usr/local/include/SDL/SDL_mixer.h -> ../../Cellar/sdl_mixer/1.2.12/include/SDL/SDL_mixer.h

**Comment by sneakernets** [comment]
That's exactly what I have. I'll recompile it and see what happens.

**Comment by sneakernets** [comment]
This was "fixed" by rebuilding SDL 1.2.15.
